swagger.yaml

```diff
@@ -3690,6 +3690,7 @@ definitions:
         - jdbc:athena
         - jdbc:bigquery
         - jdbc:databricks
+        - jdbc:dbtmetrics
```

Comment:
Not relevant to this PR

Reply:
it was created by the  bundle exec rails rake:api_docs:write script ... 🤔

Reply:
removed 👍🏿